### ISSUE TITLE
Series is cropped

### ISSUE THREAD

**Issue by jimmykane** [amcharts/amcharts4#1892]
I am sure this broke with some of the last 2 versions. It does not happen with all series though it kinda depends on the data

Example 
<img width="917" alt="Screenshot 2019-11-16 at 13 09 20" src="https://user-images.githubusercontent.com/1268723/68992957-70cd0600-0872-11ea-9650-04d59c1c9e97.png">
<img width="883" alt="Screenshot 2019-11-16 at 13 08 26" src="https://user-images.githubusercontent.com/1268723/68992958-70cd0600-0872-11ea-89a5-8850206fbf30.png">

If I set `    yAxis.extraMax = 0.20;` -> it has to be 0.20 , 0.15 does not work for example then this looks better, but still I would not like to add that extra max 

If I add the above it looks like so 

<img width="917" alt="Screenshot 2019-11-16 at 13 11 58" src="https://user-images.githubusercontent.com/1268723/68992975-b12c8400-0872-11ea-8442-721ac57ae938.png">

**I dont have this issue with the production version of the app that is running  am4charts 4.7.7** 

There it looks ok like this but the animations / parsing looks worse or delayed 

<img width="910" alt="Screenshot 2019-11-16 at 13 14 16" src="https://user-images.githubusercontent.com/1268723/68992993-05376880-0873-11ea-8cc8-eaabcccf354d.png">

**Comment by zeroin** [comment]
Hi Jimmy,

Do these charts use grouping? If yes, what groupFields do they use?

**Comment by zeroin** [comment]
Actually, if you could send us config/data would be best, as I can't reproduce it.

**Comment by jimmykane** [comment]
@zeroin  yes correct. I missed that . Grouping creates this issue. Disabling grouping works. 
```
          const screenPixes = Math.max(...[this.windowService.windowRef.screen.width, this.windowService.windowRef.screen.height]) * this.windowService.windowRef.devicePixelRatio;
          xAxis.groupCount = screenPixes

```

1440 is my width with a ratio of 2 so `2880` for grouping. 

The data are too much to post here :-/ 

However if it helps the data live on the series and not on the chart

**Comment by zeroin** [comment]
Can you send full config and data to contact@amcharts.com?

**Comment by jimmykane** [comment]
I'll try my best here. It's not easy, very complex view circles with
angular and binary stored data.

**Comment by jimmykane** [comment]
@zeroin  I think its not related per se to the grouping. 
However the latest version broke / introduced this issue. 

It also looks like the chart starts in a zoomed position. On some other series that have kinda the same issue, there is a zoom out button that fixes it.

**Comment by jimmykane** [comment]
So grouping with older than the latest version works (not sure but 4.7.7 works) but not with 4.7.10

**Comment by jimmykane** [comment]
I can confirm that the series ends up with a start state of zoomed while there is no way to unzoom @zeroin

**Comment by zeroin** [comment]
What groupField do you use for your series?

**Comment by zeroin** [comment]
Couple more questions:
1) what is the interval of your data items? is it regular?
2) when cropping occurs, is the series grouped? If yes, to what time unit?

**Comment by jimmykane** [comment]
Here are the data 

[scratch_6.txt](https://github.com/amcharts/amcharts4/files/3926999/scratch_6.txt)

**Comment by jimmykane** [comment]
Keep in mind that my chart setup follows the previous discussion we had. Series are added async to an already initialized chart

**Comment by zeroin** [comment]
This is quite a wild guess based on me knowing the source. When you set the data to the series, could you add a line `series.appeared = false`?

**Comment by jimmykane** [comment]
Adding that line did not solve it (before or after) but I am on to something. 

when the cart loads and the series starts to get drawn, half the series is drawn, up to the problematic part. Then the problematic part is drawn. 

Here is an example 

While loading the top series is already drawn but the lower is drawn half you can see. 

<img width="722" alt="Screenshot 2019-12-05 at 17 00 40" src="https://user-images.githubusercontent.com/1268723/70251960-cfeeae00-1780-11ea-9b1a-1641e72a4b8f.png">

After about 2 seconds (I suppose due to much data) the rest is drawn. 

<img width="1409" alt="Screenshot 2019-12-05 at 17 00 47" src="https://user-images.githubusercontent.com/1268723/70252030-f4e32100-1780-11ea-8ab9-6bdaf882963d.png">


Notice that only the "faulty" part is missing on the above

**Comment by zeroin** [comment]
I am actually stuck here. I was able to fix the other (#1940) min/max error really quickly just because the other user provided me with a pen where the problem was reproduced. So I am sorry, but I can only continue to investigate this issue if you could provide me with a codepen which would reproduce the issue, otherwise we can continue guessing without the end.

**Comment by jimmykane** [comment]
You are right. @zeroin  and from my side on what you do more looks like magic ;-) 

I'd put this issue aside, till I can provide more info. In the end it's a glitch with some specific data set I suppose since it happens at 2% according to the feedback I got. Nothing serious

**Comment by zeroin** [comment]
Thanks :)

**Comment by jimmykane** [comment]
Ok I have to comment here to try to help 

After many "refreshes" of the age the crop,  I notice, is not always the same. 

Timing seems related

**Comment by zeroin** [comment]
Ok, so v 4.7.14 is released.

**Comment by jimmykane** [comment]
Looks ok!

**Comment by jimmykane** [comment]
Thank you!

**Comment by zeroin** [comment]
Let's hope this problem is gone forever :)

**Comment by jimmykane** [comment]
Unfortunately found another one with the same issue :-/ 

Please dont do work here, I ll try to provide a fiddle

**Comment by zeroin** [comment]
Could you check 4.7.17?

**Comment by jimmykane** [comment]
Fixed with 4.7.17 !
